common/lib/xmodule/xmodule/js/src/capa/display.js

```diff
@@ -659,6 +659,18 @@
                         that.custom_alert("Answer submitted successfully");
                     }
 
+                    if (window.fetchCompetencyAssessmentsScore) {
+                        // This function is defined in'features/philu_courseware/competency-assessments.js(philu-edx-theme)'
+                        // which is included in courseware.html 'lms/templates/courseware/courseware.html(philu-edx-theme)'
+                        fetchCompetencyAssessmentsScore()
+                    }
+
+                    if (window.recordCompetencyAssessmentAttempt) {
+                      // This function is defined in'features/philu_courseware/competency-assessments.js(philu-edx-theme)'
+                      // which is included in courseware.html 'lms/templates/courseware/courseware.html(philu-edx-theme)'
+                      recordCompetencyAssessmentAttempt(that,response)
```

Comment:
Please properly indent these

Reply:
What's the issue with indentation @mabdullahz ?

Reply:
These 3 lines are indented with 2 spaces while the file has 4 space indentation.